Dynamic DNS docs

enolfc opened the pull request:
<!--
A good PR should describe what benefit this brings to the repository.
Ideally, there is an existing issue which the PR address.

Please check the [Contributing guide](https://docs.egi.eu/about/contributing/)
for style requirements and advice.
-->

# Summary

Documentation for the Dynamic DNS, taken from https://wiki.egi.eu/wiki/Dynamic_DNS_tutorial

<!-- Describe in plain English what this PR does -->

---

<!-- Add, if any, the related issue here, e.g. #6 -->

**Related issue :**

enolfc commented:
@tdviet, please check docs look good

gwarf commented on content/en/users/cloud-compute/dynamic-dns/_index.md:
```diff
@@ -0,0 +1,103 @@
+- NS-update portal does not store host secret in recoverable form. If you forget
+  the secret of your hostname, simply generate new one via "Show configuration"
+  button in the host edit page. The old secret will be invalid.
+
+
```
```suggestion
```

gwarf commented on content/en/users/cloud-compute/dynamic-dns/_index.md:
```diff
@@ -0,0 +1,103 @@
+  dynamic DNS, but MS Windows seems to not respect that. You can clear DNS cache
+  in Windows with `ipconfig /flushdns` command with Administrator account
+{{% /alert %}}
+
```
```suggestion
```

gwarf commented on content/en/users/cloud-compute/dynamic-dns/_index.md:
```diff
@@ -0,0 +1,103 @@
+
+1. You can also manually edit your registered hostnames in the _Overview_ menu
+   by clicking on the hostname you'd like to manage
+
```
```suggestion
```

gwarf commented on content/en/users/cloud-compute/dynamic-dns/_index.md:
````diff
@@ -0,0 +1,103 @@
+User-Agent:
+```
+
+where:
````
```suggestion
where:

```

gwarf commented on content/en/users/cloud-compute/dynamic-dns/_index.md:
````diff
@@ -0,0 +1,103 @@
+1. From the VM you'd like to assign the name to, run a command like follows:
+
+   ```shell
+   curl https://<hostname>:<secret>@nsupdate.fedcloud.eu/nic/update"
````
```suggestion
   curl "https://<hostname>:<secret>@nsupdate.fedcloud.eu/nic/update"
```

gwarf commented on content/en/users/cloud-compute/dynamic-dns/_index.md:
```diff
@@ -0,0 +1,103 @@
+
+- NS-update server uses HTTPS protocol, hostname/secret are encrypted as data
+  and not visible during transfer so it is secure to use the update URL
+
```
```suggestion
```

gwarf reviewed (changes requested):
A few minor fixes.
It may be nice to have a hint about using this from `cloud-init`.

A bit unrelated to the doc itself another point would be to discuss the static part of the DNS: `fedcloud-tf.fedcloud.eu` seems a bit long/duplicated.

enolfc commented:
> A few minor fixes.

incorporated

> It may be nice to have a hint about using this from `cloud-init`.

yup, added it also

> 
> A bit unrelated to the doc itself another point would be to discuss the static part of the DNS: `fedcloud-tf.fedcloud.eu` seems a bit long/duplicated.

Agree. To keep the current schema we could use `vm.fedcloud.eu` instead. It's short enough and does not feel repetitive.
